github-event/handler.go

```diff
@@ -238,3 +268,56 @@ func forward(req []byte, function string, headers map[string]string) (string, in
 
 	return string(body), res.StatusCode, err
 }
+
+func validCustomer(customers []string, owner string) bool {
+	found := false
+	for _, customer := range customers {
+		if len(customer) > 0 &&
+			strings.EqualFold(customer, owner) {
+			found = true
+			break
+		}
+	}
+
+	return found
+}
+
+// getCustomers reads a list of customers separated by new lines
+// who are valid users of OpenFaaS cloud
+func getCustomers(customerURL string) ([]string, error) {
```

Comment:
How is this different from what we do for Derek? Can you take a look and compare?